source/Nevermore/Mapping/IdColumnMapping.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Nevermore.Mapping
+{
+    public class IdColumnMapping : ColumnMapping, IIdColumnMappingBuilder
```

Comment:
This is a new inherited version of `ColumnMapping`, specific to the Id column, allowing for specific Id column mapping